website/docs/d/alert_configurations.html.markdown

```diff
@@ -64,11 +64,124 @@ In addition to all arguments above, the following attributes are exported:
+* `notifications` - List of notifications to send when an alert condition is detected. See [notifications](#notifications).
 * `output` - Requested output string format for the alert configuration
 
+  -> ***IMPORTANT:*** Event Type has many possible values. Details for both conditional and metric based alerts can be found by selecting the tabs on the [alert config page](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Alert-Configurations/operation/createAlertConfiguration) and checking the latest eventTypeName options.
+
+  -> **NOTE:** If `event_type` is set to `OUTSIDE_METRIC_THRESHOLD` or `OUTSIDE_SERVERLESS_METRIC_THRESHOLD`, the `metric_threshold_config` field must also be configured.
+
+### Matchers
+Rules to apply when matching an object against this alert configuration. Only entities that match all these rules are checked for an alert condition. You can filter using the matchers array only when the eventTypeName specifies an event for a host, replica set, or sharded cluster.
+
+* `field_name` - Name of the field in the target object to match on.
+
+| Host alerts         | Replica set alerts  |  Sharded cluster alerts |
+|:--------------      |:-------------       |:------                 |
+| `TYPE_NAME`         | `REPLICA_SET_NAME`  | `CLUSTER_NAME`          |
+| `HOSTNAME`          | `SHARD_NAME`        | `SHARD_NAME`            |
+| `PORT`              | `CLUSTER_NAME`      |                         |
+| `HOSTNAME_AND_PORT` |                     |                         |
+| `REPLICA_SET_NAME`  |                     |                         |
+
+
+  All other types of alerts do not support matchers.
+
+* `operator` - If omitted, the configuration is disabled.
+* `value` - If omitted, the configuration is disabled.
+
+
+* `operator` - The operator to test the field’s value.
+  Accepted values are:
+    - `EQUALS`
+    - `NOT_EQUALS`
+    - `CONTAINS`
+    - `NOT_CONTAINS`
+    - `STARTS_WITH`
+    - `ENDS_WITH`
+    - `REGEX`
+
+* `value` - Value to test with the specified operator. If `field_name` is set to TYPE_NAME, you can match on the following values:
+    - `PRIMARY`
+    - `SECONDARY`
+    - `STANDALONE`
+    - `CONFIG`
+    - `MONGOS`
+
+### Metric Threshold Config
+The threshold that causes an alert to be triggered. Required if `event_type_name` : `OUTSIDE_METRIC_THRESHOLD` or `OUTSIDE_SERVERLESS_METRIC_THRESHOLD`.
+
+* `metric_name` - Name of the metric to check. The full list being quite large, please refer to atlas docs [here for general metrics](https://docs.atlas.mongodb.com/reference/alert-host-metrics/#measurement-types) and [here for serverless metrics](https://www.mongodb.com/docs/atlas/reference/api/alert-configurations-create-config/#serverless-measurements)
+
+* `operator` - Operator to apply when checking the current metric value against the threshold value.
```

Comment:
```suggestion
* `operator` - The operator to apply when checking the current metric value against the threshold value.
```
Adding the article here for consistency because you've done that in other parts of the PR.